client/src/app/core/ui-services/media-manage.service.ts

```diff
@@ -43,6 +53,14 @@ interface ManagementPayload {
     providedIn: 'root'
 })
 export class MediaManageService {
+    public get logoSettings(): string[] {
```

Comment:
The same naming should be adopted here. (I wonder that the client was inconsistent with the naming before: A mix of `action` and `setting`?)